change/@microsoft-teams-js-5f6c016e-96a3-464a-95e1-67d1f38531b0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Defer auth window to host.",
```

Comment:
>Defer auth window to host. [](http://example.com/codeflow?start=14&length=26)

Suggestion:

Updated `authentication.authenticate` method to defer authentication requests in web hosts to the host, rather than handling internally (requests on all other types of hosts were already being deferred)

Reply:
Updated.